shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -952,4 +978,55 @@ TEST_F(AccessibilityBridgeTest, Actions) {
   EXPECT_EQ(accessibility_delegate_.actions.back(),
             std::make_pair(0, flutter::SemanticsAction::kDecrease));
 }
+
+TEST_F(AccessibilityBridgeTest, InspectData) {
+#if !FLUTTER_RELEASE
```

Comment:
Lets wrap the entire TEST_F inside of the #if

Otherwise, in release mode you have an empty TEST_F which passes trivially